ibridges/util.py

```diff
@@ -160,9 +163,34 @@ def calc_checksum(filepath: Union[Path, str, IrodsPath]):
     """
     if isinstance(filepath, IrodsPath):
         return filepath.checksum
-    f_hash=sha256()
+    if checksum_type == "sha2":
+        f_hash = sha256()
+    else:
+        f_hash = md5()
     memv=memoryview(bytearray(128*1024))
     with open(filepath, 'rb', buffering=0) as file:
         for item in iter(lambda : file.readinto(memv), 0):
             f_hash.update(memv[:item])
+    if checksum_type == "md5":
+        return f"md5:{f_hash.hexdigest()}"
```

Comment:
The native iRODS checksum in the iCAT not use the prefix `md5:` but just returns the md5sum. So will this work?

Reply:
Well, no then it's not going to work. Can we test md5 checksums on both YoDa and the native iRODS integration test?